operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -7,6 +7,35 @@
 # the pod should be restarted. The script checks a WebLogic Server state file which
 # is updated by the node manager.
 
+function newCopySitCfg() {
```

Comment:
code comment that describes function?
name of function leaves something to be desired (this was an ad hoc name).

Reply:
I have not changed any of what you wrote, not being comfortable with shell scripting.

Reply:
> > > It's always possible for a configuration change to cause pods to roll.
> 
> > I don't understand. This isn't true before your change. Before your change, a 're-instrospect'
> > in FromModel only occurred when restartVersion changed. A 'configuration change' therefore
> > does not always cause a roll.
> 
> I didn't says that a configuration change always causes a roll; I said that it is always _possible_ for one to change a roll. That is, the change can affect the definition of a pod.
> 
> > Now, does it occur when when introspectVersion changes? It seems necessary.
> 
> No, all that does is trigger an introspect. If the introspect changes the configuration of a pod, that pod will roll.

The question is about whether FromModel + introspectVersion + WebLogic configuration changes will lead to a roll.  Please do not assume pod configuration changes in the context of this question.

Reply:
For example, name this "copySitCfgLive" and rename original function "copySitCfgStart".

For inspiration on comment contents, see comment from the original function in "startServer.sh".  Also add to both Live and Start something like "Note: this function is similar to copySitCfgXXXX in xxxx.sh, but with different error handling."

Reply:
Can you add a commit that does that?